src/perform_step/ssprk_perform_step.jl

```diff
@@ -111,6 +111,74 @@ end
   integrator.u = u
 end
 
+function initialize!(integrator,cache::KYKSSPRK52Cache)
+  @unpack k,fsalfirst = cache
+  integrator.fsalfirst = fsalfirst
+  integrator.fsallast = k
+  integrator.kshortsize = 1
+  resize!(integrator.k, integrator.kshortsize)
+  integrator.k[1] = integrator.fsalfirst
+  integrator.f(integrator.fsalfirst,integrator.uprev,integrator.p,integrator.t) # FSAL for interpolation
+end
+
+@muladd function perform_step!(integrator,cache::KYKSSPRK52Cache, repeat_step=false)
+  @unpack t,dt,uprev,u,f,p = integrator
+  @unpack k,tmp, fsalfirst = cache
+  @unpack α20, α21, α30, α32, α40, α41, α43, α50, α51, α52, α54, β10, β21, β30, β32, β40, β41, β43, β50, β51, β52, β54, c1, c2, c3, c4 = cache.tab
+
+  δ = fsalfirst
+  # u1 -> stored as u
+  @.. u = uprev + dt*β10*δ
+  f(k, u, p, t+c1*dt)
+  # u2
+  @.. tmp = α20*uprev + α21*u + dt*β21*k
```

Comment:
uprev should only ever be multiplied by 1?

Reply:
While there are α10 and α20, I think uprev should be multipled by α10？

Reply:
clip the relevant part of the paper.